api/renderers.py

```diff
@@ -13,8 +13,8 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
jeAwKQnnE9VN6Vw7VNNyzIZd4YKqQScvwPSJoVj/sw2XHAbHkt/DLqpu8F5SV9QXleUBSPrgDTvSjTfObgtspqY4GSRCfPFSKl6jCSOO1gI+VuBzZnNiOqEBpaBVy9cfAecdS288i04shWiapo5gut4Acspu607M5SoJbgMHK2UW2Qn7G5pEh1x5m1qZQFuYXHcKGtOFyQjyscoumT8KSM1Y39DAlaIhG1vxDDLNQj5/IujkfXPRumlK0LxVuArZm2j+MkELPWa8tDdB1ffDfvfhJe/DpVRiGqKUzU3smarA9O+l0522gCZ03KLT4MBGXOwCkFgC5TYFej0S4Xh0wu8FPBQxkp2ku7PjUTUU5zD0r3l2LJn/g1MYqE/0ARHwDwJWkl0ewGIQg5zoZX5M7gr0uukT32tB3KrIH8NyNwJB6kTIiv8546Exavh3b8F594GfcuV6RINMnpPmE8zEsgdiWA4SaKwQJX7AVcySRL7N1UD0yuawGlF95l9pYUNUfcVtuREffTihdem2+q9SAsk2Vct3dwOElHSjsO7UQi6MxbI6gCUvrXdRGICWB7C06Htbshdr1B1wp7Z0W74vz21Crz5Q9j/KozuhWKH0pxFrJhtkdIWg0mDKHZX4lICmvZzulXLmjidB3DSiNeRgyf3P/3HL/76aO1IW9jJGW/Etalo4sw5F3zm4j9Z2DEPHajuQ+c0S0UFByADF5KILxD8MfB2NXLm712qKg9evAIKFybhQipfcWKfkwNY2pENbYPh/bDx68pX5krmInHCdXHwZpHpRQKYcc2oyj5zG1LoTmH9zApbbz/5g70PbTr8zHRjcm7bvLV9F3/wrA8vaU8sWRo+caUtA2QMRDfu6fVPTHp9PYg5bkuTXLrULyDVLROxNUCMQdvFUHwnsP4T2seclMBCeI0fhY33w+CLUOoFHjgm3VxXHNNIm551OIOLW5GX15+aH7dsUNC0cZm58aG+IhycbGwffr0aTfcW+s+Yu7ScK0zVU1sVmq2LnPnrXsFtOmXMbbZ2ojqWAHMbrLEF3rpLMQXhXCj/NASLgJVPogt1sgFLn3oVJofh5Xtgcshkc9iizmk3uvDtARnr080RLlbmnm9VliZ2LKimmUoc2vJGTa8Sq2kthoKiljBjZGEJXdvctM0AfaKmajn9lpXf0VF/+VYIB1SoftKdkxJY+kR12PEU7HwCRE3F9QF3vDYDIp6WRk5MNG8yZ6hARG0Mc7vgGfGYuPncmZPAPqedXHCrv7kZ94B0y7V+vLRujgEhSPFlgIsUE98XdEAxidCvB9YOcHJen5MYWPpMj8PmB+HYNqiOqNhLCblLHyjWu2l44aIHzQUYJ5WaqCBwLBRnHkvJSfLIcU8iqSMGdm8vB6diypCmYkaBzeCS+frh2DdbBd4OhYXQfhME807qy3LyA8sEXkdgz9Ow90mIp6HQaeMn6aVj1NegK9SmxbhRbjNK4UYFGfGYnG6gFCbZNOYHV02Dc5GcCAXi7w3cqnT7S5MPK8BzkD8NCqdD+fLZWbabAesn0V5wcDMMbYa3n1J2S++gQBeD7gnX9TDhqy1YgxRYVlOTYYw7uLgQDcM160+xbEyBC7jVB/0p97JRI/s2BYUFUN6gUj/gs7oMLgPWTzpCwNAW2P5dGNkBxS6VGOq175SaYjjxwv2RGF3jw4U8Ova3D9CY4KglicyZdCQFjgjpTsMgsJukBJm4F27di5hWX+m9HUrrgeVOEUiduCWdf1UlgcOCtdDWA5NHYOPfAm9AWLKNEgEZHQtsaYhAys3V4E4Am+K5vituw3aM+gOrsSabyxkCOy/A6QMEerIjUjR6943QcTmw4ikFqVPTYtHTqMO2h2HvY5ZN5ST64aYRWX+FM0efIvnEOSGiaM8d5pJ2yCdg1nuSGg76NP08bh+nVKgqLoLe2xRHxXKn5pt/KgML0j7m6gRsuR/G90BcTQMvcLIcXvmFkWSNu4XnHgFrnaOzQ7adISAekHHX58ml49lFqhgTdMHc2yG/AHj5FH7jT3VgYdo4vBN2fAcqhyGagrDo2AfTHoOq4Nivx5zsuwm2cOMwHCfNdFkC4S9wH6fuQy8shs5zoe1sha1T/rQEsIxnrMHuR2D8bRjZogIvnjeDD9j5CbhAAAs7FigmDGe21MgJq2nnE5HKlzYlVNFYGksV5kHH+dC5AYL2FnmzWwlY+kQN2P8s7HkYaiNQ6FJNqpgngqU6gSikJ+Dwdh008VukJ4cR1sr8eFVVErvOg/bzoXRaS73NrQcsfSaH4Ohm2PkghDkI8qlNYk7HDOlTiRxmWtVGQ4Ud66a8Q7oIWbrDV0Sqftx+JvReAW1nnIJJXwasnxx4CaiMwFv3wtgulTZyuzkxIOIwzK/eoG4SI304zDo+rCcmQXqSJSx8P5RXQcue1gWWPtURqI7B1nugMZH2oHJSabYBOFj1EQM12w6PtKcJ0bQzrcG826DnYjJLnQGrZU/lCEwMwr5HoXYktTqh07ExmsQWUrqdxvR4aUrCbkdPXGppCSx+f/rjQau/pRmwSNpYhX3fg7FtUDmoKAnO3XIvLCfTNmLt+xQ9FkLnWTDvBgg7VbKZnQxYx3OOozD4GAy/BvVRyLdbKjVtMuaCBF5JLFVRjS7d50LfFcpWZZDKgPW/i+slHPgRVA/DsVdVnThXtrGULgcJ1VfYGFZWav610HE6BMXsPcuA9X98RASHX1DwGt8KU2+b4D3J+PKd0Hs+9JwDbYsg3529Txmw/mXwasDkXgjb06lGgFy7yv4SExWetE3DGbCycxIfnr0F2cmAlZ0MWNnJgJWd7GTAyk4GrOxkwMpOdjJgZScDVnYyYGUnOxmwspMBKzsZsLKTnQxY2cmAlZ0MWNnJTgas7GTAyk4GrOxkJwNWdjJgZScDVnaykwErOxmwspMBKzvZyYCVnQxY2cmAlZ3sZMDKTgas7GTAyk52/mXnfwkwAHkfPlEQWHwkAAAAAElFTkSuQmCC"
```

Comment:
question: Should I also add the un-encoded image somewhere?

Reply:
Good idea. And maybe throw a comment in here to direct people where this came from.

Reply:
Will do.

Reply:
Added in ab6879f88888b404668fefd6bd10978c21a83f6c